### PR TITLE
Add site switcher to SUI

### PR DESCRIPTION
**Pull request by dgutride**
This adds a new site switcher component to the navigation header for SUI.  With the new home screen on the way, this paves the way for switching between multiple UI's within MiQ.  Initially, only the main administration UI will be present in this dropdown and then the home UI can be quickly added to the sites array.

The old jump icon was removed as well.
https://www.pivotaltracker.com/n/projects/1914499/stories/137176143

@serenamarie125  @beanh66 @chriskacerguis

### CODE REVIEW

**Comment by chriskacerguis** [comment]
@dgutride can you add screenshots?

**Comment by dgutride** [comment]
@chriskacerguis  - sure, this shows the new icon inline and the expanded selector
sui-screenshot

**Comment by chriskacerguis** [comment]
Nice!

**Comment by chriskacerguis** [comment]
@AllenBW could you please review?

**Comment by AllenBW** [comment]
:eyes:

File: gulp/config.js [hunk 1]
```diff
@@ -121,6 +121,7 @@ module.exports = (function() {
         './node_modules/bardjs/bard.js',
         './node_modules/sinon/pkg/sinon.js',
         './node_modules/karma-read-json/karma-read-json.js',
+        './node_modules/manageiq-ui-components/dist/js/ui-components.js',
```
Comment (AllenBW) [inline gulp/config.js]: Not needed anymore, but no action required, we'll kill this soon enough.

File: package.json [hunk 2]
```diff
@@ -162,6 +162,7 @@
     "express": "4.14.0",
     "http-proxy": "1.16.2",
     "jshint-json": "1.0.0",
+    "manageiq-ui-components": "^0.0.11",
```
Comment (AllenBW) [inline package.json]: Action required: if ya don't mind, keeping this absolute... :Heart:

**Pull request merged**